packages/config/src/ConfigAutoStore.ts

```diff
@@ -80,17 +81,11 @@ export class ConfigAutoStore {
 
         if (profile == null || profileType == null) {  // Profile must exist and have type defined
             return;
-        } else if (profileType === "base") {
-            if (profile.tokenType == null) {  // Base profile must have tokenType defined
-                return;
-            }
-        } else {
-            if (profile.basePath == null) {  // Service profiles must have basePath defined
-                return;
-            } else if (profile.tokenType == null) {  // If tokenType undefined in service profile, fall back to base profile
-                const baseProfileName = ConfigUtils.getActiveProfileName("base", opts.cmdArguments, opts.defaultBaseProfileName);
-                return this._findAuthHandlerForProfile({ ...opts, profilePath: config.api.profiles.expandPath(baseProfileName) });
-            }
+        } else if (profileType === "base" && profile.tokenType == null) {  // Base profile must have tokenType defined
+            return;
+        } else if (profile.tokenType == null) {  // If tokenType undefined in service profile, fall back to base profile
```

Comment:
🤔 not really intentional, it was more intended to prevent a confusing error with the auth-login/logout and config-secure commands.
I'll have to think that through 🤔

Reply:
After discussing IRL, this seems ok since "basePath" may not technically be required for service profiles that connect thru the API ML.

Reply:
after further testing, it would seem that this could cause issues in an environment where direct-to-service profiles force a login-prompt on Zowe Explorer.